shared/menubar/index.render.desktop.js

```diff
@@ -137,7 +137,7 @@ const FolderRow = props => {
 }
 
 const FolderEntryRow = props => {
-  const {key, entry} = props
```

Comment:
you actually can't read key from props, its not a real prop so its always undefined. Also you don't need to pass it to the children if its just going to be used in the root (which this code does) so  we can just leave it in the <FolderEntryRow key={key}> above. If you do end up needing to pass the key down for some reason you can pass it as some other named prop